storage/redis/peer_store.go

```diff
@@ -659,51 +649,62 @@ func (ps *peerStore) ScrapeSwarm(ih bittorrent.InfoHash, af bittorrent.AddressFa
 	return
 }
 
+func (ps *peerStore) collectGarbageInner(conn redis.Conn, cutoffUnix int64, swarmKeys []string) error {
+	for _, swarmKey := range swarmKeys {
+		// TODO doing this gives us safety, but might cost performance?
+		infoHash, _, isSeeder := deconstructSwarmKey(swarmKey)
+
+		// list all (peer, modified time) pairs for the infohash
+		// TODO use SCAN for this too?
+		peerList, err := redis.Strings(conn.Do("HGETALL", swarmKey))
+		if err != nil {
+			log.Error("storage: HGETALL failed", log.Fields{
+				"key":   swarmKey,
+				"error": err,
+			})
+			return err
+		}
+
+		var peerKey serializedPeer
+		//var removedPeerCount int64
+		for index, field := range peerList {
+			if index%2 == 1 { // value
+				mtime, err := strconv.ParseInt(field, 10, 64)
+				if err != nil {
+					return err
+				}
+				if mtime <= cutoffUnix {
+					log.Debug("storage: deleting peer", log.Fields{
+						"infoHash": infoHash.String(),
+						"isSeeder": isSeeder,
+						"peer":     decodePeerKey(peerKey).String(),
+					})
+
+					_, err := ps.scripts.gcPeer.execute(conn, decodePeerKey(peerKey), mtime, infoHash, isSeeder)
+					if err != nil {
+						log.Error("storage: gcPeer script execution failed", log.Fields{
+							"error": err,
+						})
+						return err
+					}
+
+				}
+			} else { // key
+				peerKey = serializedPeer([]byte(field))
+			}
```

Comment:
I think you should put this as the first case, as it is small. Then make the `// key` and `// value` comments full sentences.